elementary/monitor/data_monitoring.py

```diff
@@ -91,7 +92,7 @@ def __init__(
         self.send_test_message_on_success = send_test_message_on_success
         self.disable_samples = disable_samples
 
-    def _parse_emails_to_ids(self, emails: List[str]) -> str:
+    def _parse_emails_to_ids(self, emails: Union[str, List[str]]) -> str:
```

Comment:
Shouldn't that logic be in `SlackClient` or `AlertsAPI` rather than `DataMonitoring`?